vitest.config.ts

```diff
@@ -13,5 +13,6 @@ export default defineConfig({
     name: 'unit',
     setupFiles: ['./test/setup.ts'],
     exclude: [...defaultExclude, '**/svelte-scoped/**'],
+    globals: true,
```

Comment:
@antfu is it fine? Or do you know any better option how to run eslint tests?
Without this option no tests are found in the eslint-plugin package